docs/python/README.md

```diff
@@ -0,0 +1,378 @@
+
+OCI images are represented as sequential modifications to disk. By scoping buildpack disk modifications to their own directory, the CNB API guarantees that changes to a layer in one buildpack will not affect the contents of disk to another layer. This means that OCI images produced by CNBs are rebaseable by default, while those produced by Dockerfile are not.
+
+We saw before how the image booted a web server by default. This is accomplished using an entrypoint. In another terminal outside of the running container you can view that entrypoint:
+
+```
+$ docker inspect my-image-name | grep '"Entrypoint": \[' -A2
+            "Entrypoint": [
+                "/cnb/process/web"
+            ],
+```
+<!-- STOP. This document is autogenerated. Do not manually modify. See the top of the doc for more details. -->
+
+From within the image, you can see that file on disk:
+
+```
+$ docker run --rm my-image-name "ls /cnb/process/"
+web
+```
+<!-- STOP. This document is autogenerated. Do not manually modify. See the top of the doc for more details. -->
+
+While you might not need this level of detail to build and run an application with Cloud Native Buildpacks, it is useful to understand how they’re structured if you ever want to write your own buildpack.
+## Try CNBs out on your application
+
+So far we've learned that CNBs are a declarative interface for producing OCI images (like docker). They aim to be no to low configuration and once built, you can interact with them like any other image.
+
+For the next step, we encourage you to try running `pack` with the Heroku builder against your application and let us know how it went. We encourage you to share your experience by [opening a discussion](https://github.com/heroku/buildpacks/discussions) and walking us through what happened:
+
+- What went well?
+- What could be better?
+- Do you have any questions?
+
+We are actively working on our Cloud Native Buildpacks and want to hear about your experience. The documentation below covers some intermediate-level topics that you might find helpful.
+## Configuring your web process with the Procfile
+
+Most buildpacks rely on existing community standards to allow you to configure your application declaratively. They can also implement custom logic based on file contents on disk or environment variables present at build time.
+
+The `Procfile` is a configuration file format that was [introduced by Heroku in 2011](https://devcenter.heroku.com/articles/procfile), you can now use this behavior on your CNB-powered application via the `heroku/procfile`, which like the rest of the buildpacks in our builder [is open source](https://github.com/heroku/buildpacks-procfile). The `heroku/procfile` buildpack allows you to configure your web startup process.
+
+This is the `Procfile` of the getting started guide:
+
+```
+web: gunicorn --config gunicorn.conf.py gettingstarted.wsgi
```

Comment:
The Python getting started guide `Procfile` has a bunch of commented out lines for use by later steps of the tutorial. To keep things cleaner, I would prefer to `| head -n 1` this, so we only show the `web` process. (Which would work with the description above, since it only mentioned the `web` process.)

This is what I do for the Fir guide:
https://github.com/heroku/languages_fir_docs/blob/de485d79fda25c651b5ab07c9c034e874edff004/src/python/procfile.md?plain=1#L8

Reply:
I'm now explicitly only showing the `web` line https://github.com/heroku/buildpacks/pull/63/commits/8a16984dbda909988e128eb57a2f02fc2755ab04.